recipes/qgeoview/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class QGeoViewConan(ConanFile):
+    name = "qgeoview"
+    description = "QGeoView is a Qt / C ++ widget for visualizing geographic data. "
+    license = "LGPL-3.0"
+    topics = ("geo", "geospatial", "qt")
+    homepage = "https://github.com/AmonRaNet/QGeoView"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        self.options.rm_safe("fPIC")
+        # FIXME: we shouldn't have to force shared in qt, but config file
+        #        generated by conan in qt static is likely broken, or maybe env vars.
```

Comment:
If this is a bug, could you report an issue to the client/this repo, depending on where you feel it fits better? Thanks!

Reply:
I cannot build any qt gui library with qt static and conan v1. It's an old problem that was fixed by v2.